### DISCUSSION TITLE
Why my repository is disabled?

### DISCUSSION THREAD

**Discussion by WBCode3** [community/community#9699] [Codespaces]
![스크린샷, 2022-01-06 22-50-09](https://user-images.githubusercontent.com/80467790/148396522-caf683a1-61fd-488e-8258-cde48791871c.png)
![스크린샷, 2022-01-06 23-19-34](https://user-images.githubusercontent.com/80467790/148397274-142363d2-9a04-4595-9106-ba5cb9662343.png)

I'm selected codespace repository but my repository is all disabled
And I click code drop I'm not found "new codespace" or "open with codespace"
do you know how to solve

**Comment by dmilburn** [comment]
Hi from the Codespaces team! Codespaces is only available for access to individuals outside the context of an organization if you're part of the beta. It doesn't look like you're signed up for the beta, you can sign up [here](https://github.com/features/codespaces/signup)!

You're able to see these forms without individual access because one of the organizations you belong to allows you to create codespaces.
